src/app/SOCestimator/SOCestimator.c

```diff
@@ -0,0 +1,134 @@
+#include <stdbool.h>
+
+#include "common_macros.h"
+#include "BatteryCharacteristics.h"
+#include "SOCestimator.h"
+
+#define CELL_CAPACITY_Ah 6.550
+#define BATTERY_CAPACITY_Ah (NUM_SERIES_CELLS * CELL_CAPACITY_Ah)
```

Comment:
I think this should be the number in parallel * capacity. Bc if you have two 6Ah cells in series, and you draw 3 amps they both run out at the same time.
https://batteryguy.com/kb/wp-content/uploads/2016/01/ah-battery-banks-cheat-sheet.pdf